apis/python/src/tiledbsoma/sparse_nd_array.py

```diff
@@ -154,260 +159,150 @@ def nnz(self) -> int:
             ).nnz(),
         )
 
-    def read_sparse_tensor(
+    def read(
         self,
-        coords: SparseNdCoordinates,
-        *,
-        format: Literal["coo", "csr", "csc"] = "coo",
-    ) -> Iterator[Union[pa.SparseCOOTensor, pa.SparseCSCMatrix, pa.SparseCSRMatrix]]:
-        """
-        Read a use-defined slice of the SparseNDArray and return as an Arrow sparse tensor.
```

Comment:
It appears I've missed it -- I'm not seeing where all this great docstring content went to ... was it somehow omitted from this PR, or did it go to somewhere I missed seeing it?

Reply:
A brain fault - will fix.